### DISCUSSION TITLE
Error when I update a Chart Series value

### DISCUSSION THREAD

**Discussion by sergio-poliedro** [ShapeCrawler/ShapeCrawler#514] [General]
Hi,

I'm updating the value of a Chart Series but when I create a MemoryStream of the presentation I receive an error:

![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/118997513/7a9a2cd1-c027-4449-8c62-cc52927ba3f1)
![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/118997513/9f0e88ee-cc9b-493d-a607-a25d95467cab)

I also get the same error if I try to add a new slide before I create the MemoryStream:

![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/118997513/df2560e2-80dd-4332-95e1-8a1f2068d1f4)

Is there any workaround? Thanks in advance.

**Comment by AdamShakhabov** [comment]
Hello @sergio-poliedro,

Thanks for the bug report👍 I managed to reproduce the issue.

Unfortunately, there is no workaround right now. But, I believe a quick fix is possible. I'm on it already.

**Reply by AdamShakhabov** [reply]
Done. Check it [**0.45.2-beta.1**](https://github.com/ShapeCrawler/ShapeCrawler/files/11604505/ShapeCrawler.0.45.2-beta.1.zip)

**Reply by sergio-poliedro** [reply]
Thanks for the quick reply! 

It's working only if I update one slide. If I try to add another slide on the next iteration, I'm getting the same error. 

![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/118997513/b631e302-7f52-4a82-96ed-1f077ddf8b04)

The way my program works, is that I open a template presentation, I copy the [1] slide and update it. Then I copy again the [1] slide and update it. I do this n times. If the n = 1, with the fix you just shared it's working perfect. But if n > 1, I'm getting the error when I copy the slide on iteration #2.

**Reply by AdamShakhabov** [reply]
What about now: [**0.45.2-beta.2**](https://github.com/ShapeCrawler/ShapeCrawler/files/11618673/ShapeCrawler.0.45.2-beta.2.zip)

**Reply by sergio-poliedro** [reply]
Now it works! Thanks for the help!! :)
